docker/configure_workers_and_start.py

```diff
@@ -631,14 +632,23 @@ def main(args: List[str], environ: MutableMapping[str, str]) -> None:
         with open(mark_filepath, "w") as f:
             f.write("")
 
+    # Lifted right out of start.py
+    jemallocpath = "/usr/lib/%s-linux-gnu/libjemalloc.so.2" % (platform.machine(),)
+
+    if os.path.isfile(jemallocpath):
+        environ["LD_PRELOAD"] = jemallocpath
```

Comment:
This looks a little bit dodgy since I think we are actually modifying `os.environ` here. But for a fire-and-forget script like this I think this is fine.

Reply:
At worst, it's picking up any changes set in this one file and start.py. Everything else it does is a read. In this file, it:
1. Sets the HTTP port correctly for the reverse proxy.
2. Explicitly disables TLS, because nginx is handling it.